src/leaflet-vector-markers.scss

```diff
@@ -14,6 +14,10 @@ Version: 0.0.6
   display: block;
   text-align: center;
 
+  svg {
+    position: static !important;
```

Comment:
Hello,
What should be the change in the file leaflet.vector-marker.css to get it working correct?